integrations/openai/src/databricks_openai/agents/mcp_server.py

```diff
@@ -0,0 +1,120 @@
+from contextlib import AbstractAsyncContextManager
+from typing import Any
+
+import mlflow
+from agents.mcp import MCPServerStreamableHttp, MCPServerStreamableHttpParams, ToolFilter
+from anyio.streams.memory import MemoryObjectReceiveStream, MemoryObjectSendStream
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.credentials_provider import OAuthCredentialsProvider
+from databricks_mcp import DatabricksOAuthClientProvider
+from mcp.client.session import MessageHandlerFnT
+from mcp.client.streamable_http import GetSessionIdCallback, streamablehttp_client
+from mcp.shared.message import SessionMessage
+from mcp.types import CallToolResult
+from mlflow.entities import SpanType
+
+
+class McpServer(MCPServerStreamableHttp):
```

Comment:
One QQ, can we just test to confirm there's a good error message if you try to use this with a non-streamable HTTP server?